analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/OperationProcessors/References.cs

```diff
@@ -33,5 +33,10 @@ internal static class References
             context.State[parameterReference.Parameter] is { } symbolState
                 ? context.State.SetOperationValue(context.Operation, symbolState)
                 : context.State;
+
+        public static ProgramState Process(SymbolicContext context, IFieldReferenceOperationWrapper fieldReference) =>
```

Comment:
While this is covered in terms of coverage (some other UT code passes by), it is not covered by any single assertion. We need to add UT for this to `RoslynSymbolicExecutionTest.Operations.cs`